### PR TITLE
LPS-195927 

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-195927

### CODE REVIEW

**Comment by john-co** [comment]
pushed to https://github.com/liferay-platform-experience/liferay-portal/pull/361

File: test.properties [hunk 1]
```diff
@@ -6532,9 +6531,11 @@
             (testray.main.component.name ~ "Accessibility Framework") OR \
             (testray.main.component.name ~ "Accessibility Menu") OR \
             (testray.main.component.name ~ "Clay") OR \
+            (testray.main.component.name ~ "Control Menu") OR \
             (testray.main.component.name ~ "Feature Flag") OR \
             (testray.main.component.name ~ "Instance Settings") OR \
             (testray.main.component.name ~ "Language Override") OR \
+            (testray.main.component.name ~ "Product Menu") OR \
```
[the remark this answers is not in the document]
Comment (david-gutierrez-mesa) [reply test.properties:6538]: we have [this .testcase](https://github.com/liferay/liferay-portal/blob/8686cf09fc45a9ee6d9e21c70acfcb87ded85d4d/portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/applicationdisplaytemplate/ApplicationDisplayTemplatesWithStaging.testcase#L7 )

[the remark this answers is not in the document]
Comment (john-co) [reply test.properties:6538]: This testcase doesn't look like it focuses on testing the Product Menu. This testcase looks more like it's meant to test Application Display Templates. Are you sure this is meant to be Product Menu? Did you mean to label this as "Widget Page Templates" or something else?

[the remark this answers is not in the document]
Comment (david-gutierrez-mesa) [reply test.properties:6538]: true, It makes sense.

[the remark this answers is not in the document]
Comment (david-gutierrez-mesa) [reply test.properties:6538]: https://github.com/liferay-echo/liferay-portal/pull/14503

**Pull request closed without merging**